jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
If this job is OCEAN_ANALYSIS_LETKF, then the exscript should also have the same name.
This is an EE2 detail

Reply:
@guillaumevernieres guess we got some renaming to do

Reply:
I won't bother on renaming existing jobs and scripts.  Just take care of going forward so there is less to do later.